frontend/src/components/NavBar/index.tsx

```diff
@@ -48,19 +51,31 @@ function NavBar({ classes }: NavBarProps) {
     </Grid>
   ));
 
+  const title = get(appConfig, 'title', 'PRISM');
```

Comment:
I think you should create a `header` config object within the config instead of having everything at the top level

Reply:
Thanks @ericboucher  for the correction. So, all country will have default config where the title is set to 'PRISM' like `'header': { title: 'PRISM' }`?

Reply:
So we would do something like that:

{title, subtitle, logo} = get(appConfig, 'header', {title: 'PRISM'});